static/files/sensu-doc-template.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Title of Your Doc/Guide"
+description: "Short Description of Your Doc/Guide"
+product: <!-- Sensu product here, e.g., "Sensu Core", "Sensu Enterprise", "Plugins" -->
+version: <!-- Sensu product version here, e.g., 1.0, 1.1, 2.3, 2.8, etc -->
+weight: 4 <!-- Change this to wherever you feel like the doc should live in the nav sidebar -->
+menu:
+  <!-- Full Sensu version here, e.g., sensu-core-1.0, sensu-enterprise-2.9 -->:
+    parent: api
+---    
+# Header 1
+When you write a doc and link to a site, it should look like this: `[sensuapp.org][1]`, so functionally: [sensuapp.org][1]
+
+# Header 2
+If you create code that needs highlighting, it should look like this:
+
+```
+ {{< highlight shell >}}
+ /usr/bin/whoami{{< /highlight >}}

```

Comment:
It'd be helpful to mention any gotchas with highlighting (on the same line or can I new-line it?)